modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/CheckpointPagesWriter.java

```diff
@@ -62,6 +59,13 @@ public class CheckpointPagesWriter implements Runnable {
     /** Logger. */
     private static final IgniteLogger LOG = Loggers.forClass(CheckpointPagesWriter.class);
 
+    /**
+     * Size of a batch of pages that we drain from a single checkpoint buffer at the same time.
+     *
+     * @see #drainCheckpointBuffers(ByteBuffer, Map)
+     */
+    private static final int CP_BUFFER_PAGES_BATCH_THRESHOLD = 10;
```

Comment:
I would also like to ask you to indicate why 10 was chosen instead of 5, if it is not too much trouble.

Reply:
The choice is arbitrary